GETTING_STARTED.md

```diff
@@ -12,6 +12,66 @@
 | Counselor | Github Bot |
 <hr>
 
+### :computer: Computer Setup
+
+<details>
+<summary>Step by step instructions to setup your computer ▶️</summary>
+<br>
+ 
+1. Download or update to the newest version of MacOS or Windows.
+> 🚩 If you are using Windows, it highly recommended that you use WSL. [Here's](https://docs.microsoft.com/en-us/windows/wsl/install-win10) how.  
+2. Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). If you are on Windows, you can install Git on Windows and it should come with WSL (Ubuntu). [Here's](https://docs.microsoft.com/en-us/windows/wsl/tutorials/wsl-git) how. 
+3. Ensure that you have a [Github](https://github.com/) account and make note of the email address that you used to sign up for the account.   
+4. Download [VS Code](https://code.visualstudio.com/download) for your operating system. 
+> 🚩 If you are on Windows, [here's](https://code.visualstudio.com/docs/remote/wsl#:~:text=From%20the%20WSL%20terminal%201%20Open%20a%20WSL,3%20Type%20code%20.%20...%20More%20items...%20) how to use VS Code with WSL
+5. Enable Azure Account and Azure Functions extensions in VS Code. See below: 
+
+![Azure](https://user-images.githubusercontent.com/57218090/124665871-dff5d480-de72-11eb-86f0-35c3acf07db4.PNG)
+
+Finally, you are ready for the setup script. Follow the steps listed below:
+
+### Open MAC or WSL Terminal: 
+Run: 
+```
+sudo chown -R $(whoami) /usr/local/var/homebrew
```

Comment:
This does assume homebrew is installed which is not always the case

Reply:
@blueboxes 
[updated](https://github.com/bitprj/Intro-To-Serverless/pull/319/files#diff-d0243df16bd9ebcc544cb26394cf6ec5cc2be67bdca0a44c51457c0baf94b1c8) with specific instructions for homebrew, curl, etc.